src/main/java/org/skriptlang/skript/bukkit/log/runtime/BukkitRuntimeErrorConsumer.java

```diff
@@ -0,0 +1,128 @@
+package org.skriptlang.skript.bukkit.log.runtime;
+
+import ch.njol.skript.localization.ArgsMessage;
+import ch.njol.skript.log.SkriptLogger;
+import ch.njol.skript.util.Utils;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+import org.skriptlang.skript.log.runtime.ErrorSource;
+import org.skriptlang.skript.log.runtime.ErrorSource.Location;
+import org.skriptlang.skript.log.runtime.Frame.FrameOutput;
+import org.skriptlang.skript.log.runtime.RuntimeError;
+import org.skriptlang.skript.log.runtime.RuntimeErrorConsumer;
+
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.stream.Collectors;
+
+/**
+ * An implementation for Bukkit based platforms.
+ * Sends errors to console with formatted colours and a notification chat message to
+ * users with the see_runtime_errors permission.
+ */
+public class BukkitRuntimeErrorConsumer implements RuntimeErrorConsumer {
+
+	public final static String ERROR_NOTIF_PERMISSION = "skript.see_runtime_errors";
+
+	public static final ArgsMessage ERROR_SKIPPED = new ArgsMessage(RuntimeError.CONFIG_NODE + ".errors skipped");
+	public static final ArgsMessage ERROR_TIMEOUT = new ArgsMessage(RuntimeError.CONFIG_NODE + ".errors timed out");
+	public static final ArgsMessage ERROR_NOTIF = new ArgsMessage(RuntimeError.CONFIG_NODE + ".error notification");
+	public static final ArgsMessage ERROR_NOTIF_PLURAL = new ArgsMessage(RuntimeError.CONFIG_NODE + ".error notification plural");
+
+	public static final ArgsMessage WARNING_SKIPPED = new ArgsMessage(RuntimeError.CONFIG_NODE + ".warnings skipped");
+	public static final ArgsMessage WARNING_TIMEOUT = new ArgsMessage(RuntimeError.CONFIG_NODE + ".warnings timed out");
+	public static final ArgsMessage WARNING_NOTIF = new ArgsMessage(RuntimeError.CONFIG_NODE + ".warning notification");
+	public static final ArgsMessage WARNING_NOTIF_PLURAL = new ArgsMessage(RuntimeError.CONFIG_NODE + ".warning notification plural");
```

Comment:
These could probably go in RuntimeError too, right? What motivated the distinction?

Reply:
I forgot I had the others in RuntimeError lol

Reply:
Ah, I guess all of them could go here actually (based on our previous discussion)